src/actions/identityActions.js

```diff
@@ -240,11 +241,28 @@ export function identityWithdraw({ amountToWithdraw, withdrawTo }) {
 	return async function(dispatch, getState) {
 		try {
 			const account = selectAccount(getState())
+			const {
+				identityBalanceDai,
+				totalIdentityBalanceDai,
+			} = account.stats.formatteds
```

Comment:
is that the right place to get this (cc @IvoPaunov @rori4 )?

Reply:
Isn't there a typo here? As far as I remember those came from `account.stats.formatted`